chapter04/mdw/Ex-2.pl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/perl
+
+use 5.010;
+
+use strict;
+
+sub total{
+
+  my $total = shift @_;
+
+  foreach my $number (@_){
+
+    $total += $number
+
+  }
+
+  $total
+}
+
+my @numberlist = (1..1000);
+
+my $answer = total @numberlist;
```

Comment:
you can send the range as an argument: total( 1..1000 ) to get rid of that extra variable